Sources/Hummingbird/Utils/HTTPFields+Payload.swift

```diff
@@ -0,0 +1,14 @@
+import HTTPTypes
+
+extension HTTPFields {
+    init(contentType: String, contentLength: Int) {
```

Comment:
I wanted to centralize this as an internal helper, so that we can tweak the formula from here